### PR TITLE
More comments

### PR DESCRIPTION
**Pull request by scottleibrand**
Add more code comments giving an overview of what determine-basal is doing. Also removes some old comments and TODOs that we're never likely to do, and removes an old unused var missed in a previous cleanup.

### CODE REVIEW

**Comment by scottleibrand** [comment]
This should be a no-op, so I'll merge tomorrow. If someone can give it a quick look over before that, even better.

File: lib/determine-basal/determine-basal.js [hunk 1]
```diff
@@ -465,7 +477,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         console.error("Limiting carb impact from",ci,"to",maxCI,"mg/dL/5m (",maxCarbAbsorptionRate,"g/h )");
         ci = maxCI;
     }
-    var remainingCATimeMin = 3; // h; duration of expected not-yet-observed carb absorption
+    var remainingCATimeMin = 3; // h; minimum duration of expected not-yet-observed carb absorption
```
Comment (danamlewis) [inline lib/determine-basal/determine-basal.js]: should this comment be separated above the line to match the other comments?

**Comment by danamlewis** [comment]
LGTM other than one question about comment placement

File: lib/determine-basal/determine-basal.js (hunk 1 above)
Comment (scottleibrand) [reply lib/determine-basal/determine-basal.js]: No, this is an in-line more-detailed comment, of a style I use occasionally after short variable assignment lines etc.

**Pull request merged**